drivers/at/at.c

```diff
@@ -22,39 +23,67 @@
 #define AT_PRINT_INCOMING (0)
 #endif
 
-static void _isrpipe_write_one_wrapper(void *_isrpipe, uint8_t data)
+void _event_process_urc(event_t* _event)
 {
-    isrpipe_write_one(_isrpipe, data);
+    at_dev_t *dev = (at_dev_t*)container_of(_event, at_dev_t, event);
+    at_process_urc(dev, 1000);
+}
+
+static void _isrpipe_write_one_wrapper(void *_dev, uint8_t data)
+{
+    at_dev_t *dev = (at_dev_t*) _dev;
+    isrpipe_write_one(&dev->isrpipe, data);
+#if IS_USED(MODULE_AT_URC_ISR)
```

Comment:
Shouldn't `IS_USED` meant to be used without preprocessor `#if` ?

```suggestion
if (IS_USED(MODULE_AT_URC_ISR)) {
```

Reply:
The `event` member is not defined when the module is not present, so I can't use C conditional

Reply:
Then I think there's no need to use the `IS_USED` macro. Just a simple `#if MODULE_AT_URC_ISR` would be enough, no ?